src/s2wasm.h

```diff
@@ -935,8 +981,13 @@ class S2WasmBuilder {
       } else if (match("end_block")) {
         bstack.back()->cast<Block>()->finalize();
         bstack.pop_back();
-      } else if (match(".LBB")) {
-        s = strchr(s, '\n');
+      } else if (peek(".LBB")) {
+        // FIXME legacy tests: it can be leftover from "loop" or "block", but it can be a label too
```

Comment:
@sunfishcode: what's the status of `.LBB` and `Lfunc_end`? Looks like this PR has to introduce workarounds for them. Are they gone from `.s` now? Do we just need to update handwritten tests?